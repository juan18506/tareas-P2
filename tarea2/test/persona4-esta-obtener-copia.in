crearAgenda

crearPersona 1 20 Persona1

imprimirCopiaPersona

# creo cuatro eventos para ingresarlos a la agenda de la persona

crearFecha 22/3/2023
crearEvento 1 Descripcion uno
agregarEventoPersona

crearFecha 23/3/2023
crearEvento 2 Descripcion dos
agregarEventoPersona

imprimirCopiaPersona

crearFecha 24/3/2023
crearEvento 3 Descripcion tres
agregarEventoPersona

crearFecha 1/4/2023
crearEvento 4 Descripcion cuatro
agregarEventoPersona

imprimirCopiaPersona

# chequ√©o si existen diferentes ids de eventos y los obtengo

estaEnAgendaPersona 3
estaEnAgendaPersona 5
estaEnAgendaPersona 4
estaEnAgendaPersona 0

imprimirEventoPersona 3
imprimirEventoPersona 1

liberarPersona

Fin