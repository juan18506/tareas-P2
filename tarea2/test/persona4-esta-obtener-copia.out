1>2>3>Persona 1:  Persona1, 20 años
4>#  creo cuatro eventos para ingresarlos a la agenda de la persona.
5>6>7>8>9>10>11>Persona 1:  Persona1, 20 años
Evento 1:  Descripcion uno
Fecha: 22/3/2023
Evento 2:  Descripcion dos
Fecha: 23/3/2023
12>13>14>15>16>17>18>Persona 1:  Persona1, 20 años
Evento 1:  Descripcion uno
Fecha: 22/3/2023
Evento 2:  Descripcion dos
Fecha: 23/3/2023
Evento 3:  Descripcion tres
Fecha: 24/3/2023
Evento 4:  Descripcion cuatro
Fecha: 1/4/2023
19>#  chequéo si existen diferentes ids de eventos y los obtengo.
20>El evento con id 3 está en la agenda de la persona.
21>El evento con id 5 NO está en la agenda de la persona.
22>El evento con id 4 está en la agenda de la persona.
23>El evento con id 0 NO está en la agenda de la persona.
24>Evento 3:  Descripcion tres
Fecha: 24/3/2023
25>Evento 1:  Descripcion uno
Fecha: 22/3/2023
26>27>Fin.
